starting_package/openvino_workbench/container.py

```diff
@@ -52,27 +53,25 @@ def wait_for_stage_to_complete(pre_message: str,
     print('Done.')
 
 
-def print_finish_message(protocol: str, port: int, tokens: dict):
-    link = f'{protocol}://127.0.0.1:{port}/'
-    message = f'DL Workbench is available at {link}'
-
+def print_finishing_message(docker_client: DockerClient, container_name: str, is_detached: bool):
     print(DL_WB_LOGO)
 
-    if 'login_token' not in tokens or 'url_token' not in tokens:
-        print(message)
-        return
-
-    print(f'{message}?token={tokens["url_token"]}\n'
-          '\nNote: Authentication with the token inside this link is available only once.\n'
-          f'The link expires after you click it. Use your login token at {link} to '
-          'authenticate again.\n')
+    # Show finishing message from the container logs
+    container = docker_client.containers.get(container_name)
+    all_logs = container.logs().decode('utf-8').rstrip()
+    finishing_message_start = all_logs.rfind(WORKBENCH_READY_MESSAGE)
+    print(f'\n{all_logs[finishing_message_start:]}')
+    del all_logs
```

Comment:
?

Reply:
If we restart a container, we know nothing about the ip, port, protocol, etc. so we cannot construct a proper finishing message. On the other hand, there is a finishing message in container logs so I've decided to simplify the finishing message logic and rely on the container logs both for the first start of a container and for the restart.

Reply:
omg, that is strange